apps/src/javalab/javalabRedux.js

```diff
@@ -127,6 +129,13 @@ export const setLevelName = levelName => ({
   levelName
 });
 
+export const setDisableFinishButton = disableFinishButton => {
```

Comment:
I don't love this pattern of using redux to set whether `Finish` can be pressed or not but I couldn't think of a better way. Definitely open to suggestions here.